code/modules/reagents/reagent_containers/food.dm

```diff
@@ -189,6 +210,11 @@
 	qdel(src)
 
 /obj/item/chems/food/Destroy()
+	if(plate)
```

Comment:
this is causing some CI fails. i think when we qdel the food the plate should be lost too, and it should instead be dropped in, i don't know, physically_destroyed or eating code instead